glean-core/android/src/main/java/mozilla/telemetry/glean/scheduler/MetricsPingScheduler.kt

```diff
@@ -62,6 +62,24 @@ internal class MetricsPingScheduler(
         }
     }
 
+    /**
+     * Safely transforms a `Date` to a `String`.
```

Comment:
Not sure if worth pointing out that it uses a placeholder string in case of that exception?

Reply:
It is! Fixing it!